### DISCUSSION TITLE
How to integrate trustwallet in react.js?

### DISCUSSION THREAD

**Discussion by Sun-Shine2** [community/community#21168] [Codespaces] [answered]
Now I am building a web3 project. But I have a problem in trust wallet integration. The project is working well with Metamask. But don't work for trustwallet. How can I fix this?

**Comment by patronus5** [comment, marked as answer]
You can use Web3Modal module in react.js. You can refer to web3Modal module docuement for details.

**Reply by Sun-Shine2** [reply]
Thank you
